third_party/blink/renderer/modules/brave/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
```

Comment:
is this actually needed?

Reply:
doesn't look like it - removed and it still compiled 🤷‍♂ (although, I did an incremental) Let's see how PR builder handles it